src/Concordium/Client/Runner.hs

```diff
@@ -897,46 +898,99 @@ processTransactionCmd action baseCfgDir verbose backend =
+        let operation = CBOR.TokenTransferBuilder (Just amount) (Just tokenReceiver) maybeCborMemo
+        let eitherTokenTranfserBody = CBOR.buildTokenTransfer operation
+        tokenTransferBody <- case eitherTokenTranfserBody of
+            Right val -> return val
+            Left err -> logFatal ["Error creating token transfer body:", err]
+        let tokenHolderTransfer = CBOR.TokenHolderTransfer tokenTransferBody
+        let tokenHolderTransaction = CBOR.TokenHolderTransaction (Seq.fromList [tokenHolderTransfer])
+        let bytes = CBOR.tokenHolderTransactionToBytes tokenHolderTransaction
+        let tokenParameter = Types.TokenParameter $ BS.toShort bytes
 
-                let symbol = tokenIdFromText symbolText
+        let symbol = tokenIdFromText symbolText
 
-                let payload = Types.TokenHolder symbol tokenParameter
-                let encodedPayload = Types.encodePayload payload
+        let payload = Types.TokenHolder symbol tokenParameter
+        let encodedPayload = Types.encodePayload payload
 
-                let nrgCost _ = return $ Just $ simpleTransferEnergyCost $ Types.payloadSize encodedPayload
-                txCfg <- liftIO $ getTransactionCfg baseCfg txOpts nrgCost
+        let nrgCost _ = return $ Just $ simpleTransferEnergyCost $ Types.payloadSize encodedPayload
+        txCfg <- liftIO $ getTransactionCfg baseCfg txOpts nrgCost
 
-                let intOpts = toInteractionOpts txOpts
-                let outFile = toOutFile txOpts
-                signAndProcessTransaction_ verbose txCfg encodedPayload intOpts outFile backend
+        let intOpts = toInteractionOpts txOpts
+        let outFile = toOutFile txOpts
+        signAndProcessTransaction_ verbose txCfg encodedPayload intOpts outFile backend
+
+handlePLTUpdateSupply ::
+    Backend ->
+    Maybe FilePath ->
+    Bool ->
+    TokenSupplyAction ->
+    TokenAmount ->
+    Text ->
+    TransactionOpts (Maybe Types.Energy) ->
+    IO ()
+handlePLTUpdateSupply backend baseCfgDir verbose tokenSupplyAction amount symbolText txOpts = do
+    baseCfg <- getBaseConfig baseCfgDir verbose
+    when verbose $ do
+        runPrinter $ printBaseConfig baseCfg
+        putStrLn ""
+
+    withClient backend $ do
+        tokenGovernanceOperation <- case tokenSupplyAction of
+            Mint -> pure $ CBOR.TokenMint amount
+            Burn -> pure $ CBOR.TokenBurn amount
+
+        let tokenGovernanceTransaction = CBOR.TokenGovernanceTransaction (Seq.fromList [tokenGovernanceOperation])
+        let bytes = CBOR.tokenGovernanceTransactionToBytes tokenGovernanceTransaction
+        let tokenParameter = Types.TokenParameter $ BS.toShort bytes
+
+        let symbol = tokenIdFromText symbolText
+
+        let payload = Types.TokenGovernance symbol tokenParameter
+        let encodedPayload = Types.encodePayload payload
+
+        let nrgCost _ = return $ Just $ simpleTransferEnergyCost $ Types.payloadSize encodedPayload
```

Comment:
Probably shouldn't use this as it's tied to `simpleTransferCost`. Currently, `tokenHolderBaseCost` is used instead, though both happen to be 300.